packages/tools/env/apm/test.coffee

```diff
@@ -0,0 +1,12 @@
+
+module.exports =
+  tags:
+    posix: false
+    tools_dconf: false
+    tools_repo: false
+    tools_rubygems: false
+    tools_apm: true
```

Comment:
remove the tags marked as false,
add your new tag into "packages/tools/test.sample.coffee" as `false`

Reply:
OK on it.